packages/odyssey-react-mui/src/FieldComponentProps.ts

```diff
@@ -51,4 +55,17 @@ export type FieldComponentProps = {
    * The name of the `input` element. Defaults to the `id` if not set.
    */
   name?: string;
-};
+} & (
+  | {
+      errorMessage: string;
+      errorMessagesList?: never;
+    }
+  | {
+      errorMessage?: never;
+      errorMessagesList: string[];
+    }
+  | {
+      errorMessage?: never;
+      errorMessagesList?: never;
+    }
+);
```

Comment:
@bryancunningham-okta I decided to add the mutually exclusive props here since users will first pass in `errorMessage` and `errorMessagesList` to higher-level components that take in `FieldComponentProps` and we want the error to be raised at compile time as discussed

Reply:
I don't know if this logic holds relative to `PasswordField`. I'm wondering if we should remove it.